optional_components/embeddings/app/config.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+HUGGINGFACE_MODEL = os.environ.get("HUGGINGFACE_MODEL", "thenlper/gte-large")
+STATIC_TOKEN = os.environ.get("EMBEDDINGS_API_KEY", "add-token")
```

Comment:
Can we call the config itself API_KEY just for clarity?

Reply:
You mean instead of `EMBEDDINGS_API_KEY`. I added it to be more specific since in the future we might have many api keys.